app/services/i18n/i18n.ts

```diff
@@ -32,7 +32,7 @@ export function $t(...args: any[]) {
 const LANG_CODE_MAP = {
   cs: { lang: 'Czech', locale: 'cs-CZ' },
   de: { lang: 'German', locale: 'de-DE' },
-  en:	{ lang: 'English', locale: 'en-US' },
+  'en-US':	{ lang: 'English', locale: 'en-US' },
```

Comment:
Should be have both `en` and `en-US`?  It notice for portuguese we have both `pt` and `pt-BR`.

Reply:
I changed it only for consistency, because we have translations only for `en-US` (as a source language), at the same time we have both `pt` and `pt-BR` translations.

Reply:
Ah, got it.